gems/pending/appliance_console.rb

```diff
@@ -475,16 +475,32 @@ module ApplianceConsole
           when CANCEL
             # don't do anything
           when RE_RESTART
-            Env['RESTARTOS'] = are_you_sure?("restart the appliance now")
+            if are_you_sure?("restart the appliance now")
+              Logging.logger.info("APPLIANCE RESTART initiated by MIQ Console.")
```

Comment:
I feel like the log message should say "appliance console" as opposed to "MIQ Console"